docs/filtering_data/filtering_data_with_panpipes.md

```diff
@@ -13,7 +13,7 @@ In your previously created the `teaseq` directory, create a new folder to run th
 mkdir preprocessing && cd preprocessing
 ```
 
-In here, run `panpipes preprocess config`, which will generate again a `pipeline.yml` file for you to customize. You can review and download the yaml here: [pipeline_preprocess.yml](pipeline_yml).
+In here, run `panpipes preprocess config`, which will generate again a `pipeline.yml` file for you to customize. You can review and download the yaml here: [pipeline_preprocess.yml](pipeline_yml.md).
```

Comment:
```suggestion
In here, run `panpipes preprocess config`, which will generate again a `pipeline.yml` file for you to customize. You can review and download the preprocess `pipeline.yml` [here](pipeline_yml.md).
```

Reply:
I find it a bit confusing if the file is suddenly referred to as `pipeline_preprocess.yml` instead of just `pipeline.yml`.